plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -388,9 +390,13 @@ def _check_update_figure_dict(
         if updated_trace_indices is None:
             updated_trace_indices = []
 
+        if indices_to_use is None:
```

Comment:
So these are the indices which are used?

Reply:
By default, this argument is none, so this means that by default, no traces will be resampled? 

```python
if idx in updated_trace_indices or idx not in indices_to_use:
```